src/menus/EditMenus.cpp

```diff
@@ -354,13 +354,16 @@ std::pair<double, double> FindSelection(const CommandContext &context)
 {
    double sel0 = 0.0, sel1 = 0.0;
    
+#if 0
```

Comment:
Will this be re-enabled in future?

Reply:
That depends on designers’ choices.  If we ever really want a right click action sensitive to the exact x mouse position, it may be useful.  But that was not a correct interpretation of designers’ wishes in this case.